flask_resources/context.py

```diff
@@ -58,6 +60,15 @@ def __exit__(self, type, value, traceback):
         """Pop the resource context manager from the current request."""
         del g.resource_requestctx
 
+    def inject_original_request(self):
```

Comment:
Major: Could this be flattened? I have the feeling that:

```
self.request_args = self.original_request.args
self.request_content = deserialized(self.original_request.data)
self.accept_mimetype = self.original_request.accept_mimetype
self.content_type = self.original_request.content_type
```

In the data case, maybe we need two attributes... In general I think this could be cleaner.